src/check-dsl.ts

```diff
@@ -27,13 +27,43 @@ const defaultError = (lines: any) => {
   };
 };
 
+const indentDSL = (rawDsl: string) => {
+  const indentType = "";
+  const indentRelation = "  ";
+  const indentDefine = "    ";
+
+  return rawDsl.split("\n")
+    .filter((el: string) => el.trim().length > 0)
```

Comment:
```suggestion
    // filter out any empty lines
    .filter((line: string) => line.trim().length)
```

Prefer to use descriptive variables (e.g. `line` over `el`) when possible, future us will be thankful